src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -189,7 +189,8 @@
                     (if (contains? domain-groups config/tcga-authorization-domain)
                       tcga-disabled-text
                       non-dbGap-disabled-text))
-      :restricted? (seq auth-domain))))
+      :restricted? (seq auth-domain)
+      :featured?  (contains? featured-workspaces (select-keys workspace [:namespace :name])))))
```

Comment:
You already have `(select-keys workspace [:namespace :name])` defined as `workspace-id`